test/EasyPost/ScanFormTest.php

```diff
@@ -69,7 +69,7 @@ public function testRetrieve(ScanForm $scanform)
         $retrieved_scanform = ScanForm::retrieve($scanform->id);
 
         $this->assertInstanceOf('\EasyPost\ScanForm', $retrieved_scanform);
-        $this->assertStringMatchesFormat('sf_%s', $retrieved_scanform->id);
+        $this->assertStringMatchesFormat($scanform, $retrieved_scanform);
```

Comment:
The assertion here is wrong, it should be assertEquals()
```suggestion
        $this->assertEquals($scanform, $retrieved_scanform);
```

Reply:
Nice catch.